stack/source.go

```diff
@@ -167,6 +167,9 @@ func name(n ast.Node) string {
 	if s, ok := n.(*ast.SelectorExpr); ok {
 		return s.Sel.Name
 	}
+	if s, ok := n.(*ast.StarExpr); ok {
```

Comment:
I'd say it's now time to switch to a switch case (pun intended).

Reply:
Done!